### PR TITLE
FUCKING HELL DYLAN.

### LINKED ISSUE #5630: (WEB REPORT BY: tubbywatch REMOTE: 198.245.63.50:7777) Wonky Lokking wall-mounted lanterns
Revision edf4be5c0b5434b168a4ffe495324de9e7be00f9
>General description of the issue
The wall-mounted lantern moves to the wrong spot
What you expected to happen
It doesn't move

>What actually happened
It moves and then looks like its floating

>Steps to reproduce if possible
Take the latern off the wall-mounted lantern then put it back on

### LINKED ISSUE #5631: (WEB REPORT BY: tubbywatch REMOTE: 198.245.63.50:7777) Wallmounted lanterns
Revision edf4be5c0b5434b168a4ffe495324de9e7be00f9
General description of the issue
The wall-mounted lantern moves to the wrong spot
What you expected to happen
It doesn't move

What actually happened
It moves and then looks like its floating

Steps to reproduce if possible
Take the latern off the wall-mounted lantern then put it back on

### LINKED ISSUE #5760: After removing and replacing wall mounted lamp, the mount aligns itself towards south.
Basically what's in the title, vid related, of course in shit quality.
https://youtu.be/uXZpkaUJwiY

### PR DESCRIPTION
**Pull request by PJB3005**
Fixes #5631
Fixes #5630
Fixes #5760

### CODE REVIEW

**Comment by clusterfack** [comment]
Using undocumented features to fix a random ass bug :anguished:

**Comment by PJB3005** [comment]
:^)

File: code/modules/mining/mine_structures.dm [hunk 1]
```diff
@@ -40,7 +40,8 @@
 		"<span class='notice'>You put \a [W.name] on the \the [src].</span>")
 		playsound(get_turf(src), 'sound/machines/click.ogg', 20, 1)
 		qdel(W)
-		new /obj/structure/hanging_lantern(get_turf(src), src.dir)
+		var/obj/structure/hanging_lantern/L = new/obj/structure/hanging_lantern(get_turf(src)) // Obligatory "fuck dylan".
```
Comment (DeityLink) [inline code/modules/mining/mine_structures.dm]: by the way, you can just write "var/obj/structure/hanging_lantern/L = new(get_turf(src))"

Comment (PJB3005) [inline code/modules/mining/mine_structures.dm]: oh, yeah, true, leftover from when I used the modified type.

**Pull request merged**